src/api.jl

```diff
@@ -4,12 +4,13 @@ using ..Telegram
 include("telegram_api.jl")
 
 for (f, fdoc) in TELEGRAM_API
+    fs=string(f)
     @eval begin
         function $f(client::Telegram.TelegramClient = Telegram.DEFAULT_OPTS.client; kwargs...)
             params = Dict{Symbol, Any}(kwargs)
             params[:chat_id] = get(params, :chat_id, Telegram.chatid(client))
             params[:parse_mode] = get(params, :parse_mode, client.parse_mode)
-            Telegram.query(client, String(Symbol($f)), params = params)
+            Telegram.query(client, $fs, params = params) # interpolation string directly, much faster
```

Comment:
Maybe it can be done as a single line? Something like
```
Telegram.query(client, string($f), params = params)
```

And there is no need for this comment, those who read source code wouldn't be able to understand faster than what this string is.

Reply:
``Telegram.query(client, string($f), params = params)``
> I had tried that  , it is slower than the original `String(Symbol($f))`
   due to name conflict on function close, maybe it is a better idea to rename it to closetg
   such as  `fs = f==:close ? string(f,:tg) : string(f)`

Reply:
What do you mean by "slower"? Surely all of these transformations take less time than actual HTTP requests.

Reply:
Regarding `close` I think it's better just to add `import Base: close` at the beginning.

It's not very important anyway, since all of these functions aren't exported, so one can always write explicitly `Telegram.close`.